hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleStakingInfo.java

```diff
@@ -525,7 +525,7 @@ public MerkleStakingInfo(
 
     @Nullable
     @VisibleForTesting
-    byte[] getHistoryHash() {
+    public byte[] getHistoryHash() {
```

Comment:
Hmm, I'm not sure we want to expose this. But maybe it's not a big deal. @tinker-michaelj thoughts?

Reply:
IMO it's ok as the method is (already) annotated with `@VisibleForTesting` which to me means in this case it is acceptable to use in `cli-clients` - utility tools for programmers.  Users (e.g., this dumper) are then on notice that it could change out from under them in the future and it's on them to deal with it, while in the meantime it's totally ok for a utility tool to use.